CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # CHANGELOG
 
+## v0.25.0
+
+*December 14, 2022*
+
+This release updates the tendermint-rs dependency to v0.28.0 which includes important security improvements. Many other improvements have been made as well, including misbehaviour handling.
+
+A lot of work has also been put towards implementing ADR 5, which is currently unfinished and has been put behind the feature flag `val_exec_ctx`.
+
+The only consensus-breaking changes are the ones related to the fact that we now properly handle misbehaviour messages.
+
+### BREAKING CHANGES
+
+- Implement the IBC misbehaviour handler and misbehaviour handling logic for the Tendermint light client.
+  ([#12](https://github.com/cosmos/ibc-rs/issues/12))
+- `Ics20Context` no longer requires types implementing it to implement `ChannelReader` and `ChannelKeeper`, and instead depends on the `Ics20ChannelKeeper`
+  and `SendPacketReader` traits. Additionally, the `send_packet` handler now requires the calling context to implement `SendPacketReader` and returns
+  a `SendPacketResult`.
+  ([#182](https://github.com/cosmos/ibc-rs/issues/182))
+- Add `ValidationContext` and `ExecutionContext`, and implement for clients (ICS-2)
+  ([#240](https://github.com/cosmos/ibc-rs/issues/240))
+- Change `host_height`, `host_timestamp` return value to a `Result` in `ClientReader`, `ConnectionReader`, `ChannelReader` and `ValidationContext`
+  ([#242](https://github.com/cosmos/ibc-rs/issues/242))
+- Rename Ics* names to something more descriptive
+  ([#245](https://github.com/cosmos/ibc-rs/issues/245))
+- Implement `ValidationContext::validate` and `ExecutionContext::execute` for connections (ICS-3)
+  ([#251](https://github.com/cosmos/ibc-rs/issues/251))
+- Implement misbehaviour in `ExecutionContext` and `ValidationContext`
+  ([#281](https://github.com/cosmos/ibc-rs/issues/281))
+- Update `tendermint` dependencies to `v0.28.0`, which contain an important security fix.
+([#294](https://github.com/cosmos/ibc-rs/issues/294))
+
+### BUG FIXES
+
+- Set counterparty connection ID to None in `conn_open_init` ([#174](https://github.com/cosmos/ibc-rs/issues/174))
+- Verify the message's counterparty connection ID in `conn_open_ack`
+  instead of the store's ([#274](https://github.com/cosmos/ibc-rs/issues/274))
+
+### IMPROVEMENTS
```

Comment:
Can we move entries in `.changelog/enhancement` into `.changelog/improvements`?

Reply:
[bd29d96](https://github.com/cosmos/ibc-rs/pull/300/commits/bd29d962ada53e974f954d47d37f5a20f8fcff10)